Makefile.am

```diff
@@ -21,7 +21,7 @@ clean-local:
 
 #if SYSTEMD
 systemdsystemunitdir = @SYSTEMD_SYSTEMUNITDIR@
-systemdsystemunit_DATA = csd.service
+systemdsystemunit_DATA = dcm.service
```

Comment:
I think the service name is cryptic: **dcm**. How about `dronecode-camera-mgr.service` ? Many systemd services are self explanatory, so that the service name briefly tells what it is.

Reply:
Fixed. Renamed it to dronecode-camera-manager.service